examples/docs_snippets/docs_snippets/concepts/partitions_schedules_sensors/sensors/asset_sensors.py

```diff
@@ -16,23 +16,36 @@ def my_job():
     pass
 
 
+from typing import List
+
+from dagster import Config
+
+
+class ReadMaterializationConfig(Config):
+    asset_key: List[str]
+
+
 # start_asset_sensor_marker
-from dagster import AssetKey, EventLogEntry, SensorEvaluationContext, asset_sensor
+from dagster import (
+    AssetKey,
+    EventLogEntry,
+    RunConfig,
+    SensorEvaluationContext,
+    asset_sensor,
+)
 
 
 @asset_sensor(asset_key=AssetKey("my_table"), job=my_job)
 def my_asset_sensor(context: SensorEvaluationContext, asset_event: EventLogEntry):
     yield RunRequest(
         run_key=context.cursor,
-        run_config={
-            "ops": {
-                "read_materialization": {
-                    "config": {
-                        "asset_key": asset_event.dagster_event.asset_key.path,  # type: ignore
-                    }
-                }
+        run_config=RunConfig(  # type: ignore
+            ops={
+                "read_materialization": ReadMaterializationConfig(
+                    asset_key=list(asset_event.dagster_event.asset_key.path)  # type: ignore
```

Comment:
why all the `type: ignore`s here?

Reply:
The first is addressed by https://github.com/dagster-io/dagster/pull/13708 (and is removed now that this is stacked). The second is just that `asset_key` can be null

Reply:
I would assert or check that asset_key is not none rather than ignoring the type hint